microservices/configcheck/src/main/java/datawave/microservice/configcheck/CommandRunner.java

```diff
@@ -0,0 +1,225 @@
+package datawave.microservice.configcheck;
+
+import static datawave.microservice.configcheck.util.ArgumentUtils.OUTPUT;
+import static datawave.microservice.configcheck.util.ArgumentUtils.getFile;
+import static datawave.microservice.configcheck.util.ArgumentUtils.getFileList;
+import static datawave.microservice.configcheck.util.ArgumentUtils.getFiles;
+import static datawave.microservice.configcheck.util.ArgumentUtils.getOutputPath;
+import static datawave.microservice.configcheck.util.FileUtils.getFilePath;
+import static datawave.microservice.configcheck.util.XmlRenderUtils.loadContent;
+import static datawave.microservice.configcheck.util.XmlRenderUtils.loadProperties;
+import static datawave.microservice.configcheck.util.XmlRenderUtils.loadYamlAsProperties;
+import static datawave.microservice.configcheck.util.XmlRenderUtils.renderContent;
+import static datawave.microservice.configcheck.util.XmlRenderUtils.valueToObject;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.ApplicationArguments;
+
+public class CommandRunner {
+    private static Logger log = LoggerFactory.getLogger(ConfigCheckApplication.class);
+    
+    public static final String RENDER = "render";
+    public static final String ANALYZE = "analyze";
+    public static final String COMPARE = "compare";
+    
+    public static final String HELP = "help";
+    public static final String CONFIGDIR = "configdir";
+    public static final String PROPERTIES = "properties";
+    public static final String YAML = "yaml";
+    public static final String FULL_REPORT = "fullreport";
+    
+    private ApplicationArguments args;
+    
+    public CommandRunner(ApplicationArguments args) {
+        this.args = args;
+    }
+    
+    public String run() {
+        String output = "";
+        boolean help = args.containsOption(HELP);
+        if (!args.getNonOptionArgs().isEmpty()) {
+            switch (args.getNonOptionArgs().get(0)) {
+                case RENDER:
+                    try {
+                        if (!help) {
+                            output = runRenderCommand(args);
+                        }
+                    } catch (Exception e) {
+                        log.error("Encountered exception during render", e);
+                    } finally {
+                        if (output == null || output.isEmpty()) {
```

Comment:
If you specify the --output option, then this output will be empty and the usage displays anyway.  This was quite confusing when I was running this from the command line.  Perhaps the render command should output a statement as to the output being rendered successfully.

Reply:
that was unintended.  fixing...

Reply:
fixed